py-polars/tests/test_utils.py

```diff
@@ -53,5 +53,13 @@ def test_timedelta_to_pl_timedelta() -> None:
 
 
 def test_estimated_size() -> None:
-    a = pl.Series([1, 2, 3])
-    assert a.estimated_size() == a.to_frame().estimated_size()
+    s = pl.Series("n", list(range(10_000)))
```

Comment:
I want to keep the tests lightweight. Can test this on a bit smaller data.

Reply:
Done;  < 1ms now.